doc/source/whatsnew/v0.20.0.txt

```diff
@@ -457,11 +457,8 @@ Bug Fixes
 - Bug in ``pd.read_csv()`` with ``float_precision='round_trip'`` which caused a segfault when a text entry is parsed (:issue:`15140`)
 
 
+- Bug in ``DataFrame.boxplot`` where ``fontsize`` was not applied to the tick labels on both axes (:issue:`15108`)
 
 
 
-
-
-
-
-- Bug in ``DataFrame.boxplot`` where ``fontsize`` was not applied to the tick labels on both axes (:issue:`15108`)
+- Bug in ``resample().size()``. Inconsistent return type on resample of empty DataFrame (:issue:`14962`)
```

Comment:
just move this in the bug fix section up higher, that way you won't have conflicts